backend/FwLite/FwDataMiniLcmBridge.Tests/MiniLcmTests/UpdateEntryTests.cs

```diff
@@ -44,4 +44,57 @@ public async Task UpdateEntry_WithNullLexemeFormOA_CreatesNewLexemeForm()
         updatedEntry.LexemeForm.Should().NotBeNull();
         updatedEntry.LexemeForm["en"].Should().Be("updated test");
     }
+
+    [Fact]
+    public async Task UpdateEntry_CanUpdateExampleSentenceTranslations_WhenNoTranslationObjectExists()
+    {
+        // Arrange
+        var entry = await Api.CreateEntry(new Entry
```

Comment:
This is a pretty verbose entry considering you're testing just a single property.

Reply:
Yeah, test names in the Object_CanDoTheThing_WhenConditionExists convention do get pretty long. I'm not a big fan of that naming convention myself, but it's popular. Anyway, I'll go ahead and merge, and we can rename that later if we feel like it.